<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_data">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="TQtWidget" name="widget_root_01" native="true"/>
            </item>
            <item row="0" column="0">
             <widget class="TQtWidget" name="widget_root_00" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="TQtWidget" name="widget_root_10" native="true"/>
            </item>
            <item row="1" column="1">
             <widget class="TQtWidget" name="widget_root_11" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_files">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="0">
         <widget class="QLineEdit" name="lineEdit_pp"/>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton_load_pp">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_select_loc">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLineEdit" name="lineEdit_loc"/>
        </item>
        <item row="19" column="1">
         <widget class="QPushButton" name="pushButton_select_crystal">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_config"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_config">
          <property name="text">
           <string>System Config</string>
          </property>
         </widget>
        </item>
        <item row="19" column="2">
         <widget class="QPushButton" name="pushButton_load_crystal">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_crystal">
          <property name="text">
           <string>Crystal Energy Histograms</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLineEdit" name="lineEdit_crystal"/>
        </item>
        <item row="13" column="0">
         <widget class="QLineEdit" name="lineEdit_apd"/>
        </item>
        <item row="13" column="1">
         <widget class="QPushButton" name="pushButton_select_apd">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_apd">
          <property name="text">
           <string>APD Energy Histograms</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_flood">
          <property name="text">
           <string>Floods</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QPushButton" name="pushButton_load_apd">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLineEdit" name="lineEdit_graph"/>
        </item>
        <item row="15" column="0">
         <widget class="QLineEdit" name="lineEdit_flood"/>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_graph">
          <property name="text">
           <string>Segmentations</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="1">
         <widget class="QPushButton" name="pushButton_select_flood">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QPushButton" name="pushButton_select_graph">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QPushButton" name="pushButton_load_flood">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QPushButton" name="pushButton_load_graph">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_cal">
          <property name="text">
           <string>Calibration File</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLineEdit" name="lineEdit_cal"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_select_config">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="pushButton_select_cal">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_load_config">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="pushButton_load_cal">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_select_pp">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_loc">
          <property name="text">
           <string>Crystal Position File</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_pp">
          <property name="text">
           <string>Photopeak Peak File</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_ped">
          <property name="text">
           <string>Pedestal File</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_load_loc">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_ped"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_select_ped">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_load_ped">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_panel">
        <property name="maximumSize">
         <size>
          <width>38</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_cartridge">
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_fin">
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>M</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_module">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_apd">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TQtWidget</class>
   <extends>QWidget</extends>
   <header location="global">TQtWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
